### PR TITLE
FSET-1068: Complete scheme selection before selecting the locations

### PR DESCRIPTION
**Pull request by satishkamatam**

### CODE REVIEW

File: app/views/application/scheme/chooseyourlocations.scala.html [hunk 1]
```diff
@@ -20,14 +20,29 @@
...
+@main_template(title = "Choose your preferred locations?", pageForms = Seq(locationForm), None, List("location-finder.js")) {
 
     @helper.form(action = routes.SchemeController.submitLocations(), 'novalidate -> "novalidate") {
         @helper.CSRF.formField
-        <h1 class="heading-xlarge">Where could you work from?</h1>
-
         <input type="text" id="hiddenInput" class="visuallyhidden">
         <div class="grid-wrapper">
+            <div class="grid grid-2-3">
+                <h1 class="heading-xlarge" id="choosePrefLocFramHeading">Choose your preferred locations</h1>
```
Comment (henricook) [inline app/views/application/scheme/chooseyourlocations.scala.html]: It's only a typo in an ID.. but should this be ..."FromHeading"...? or "FormHeading"?

Comment (satishkamatam) [reply app/views/application/scheme/chooseyourlocations.scala.html]: Hmm taken from prototype as is and hence the typo .. let me remove id if not required

File: app/views/application/scheme/chooseyourschemes.scala.html [hunk 2]
```diff
@@ -49,7 +55,7 @@ <h1 class="heading-xlarge" xmlns="http://www.w3.org/1999/html">Choose your prefe
                                 <label for="scheme-@scheme.id" class="block-label block-label-slim">
                                     <input type="checkbox" id="scheme-@scheme.id" data-schemename value="@scheme.id">@scheme.name</input>
                                 </label>
-                                <span class="alwayshidden" data-scheme-req>@schemeRequirements(scheme)</span>
+                                <span class="alwayshidden" @if(isSpecialScheme(scheme)){data-spec-scheme} else {data-scheme-req}>@schemeRequirements(scheme)</span>
```
Comment (henricook) [inline app/views/application/scheme/chooseyourschemes.scala.html:58]: What's a special scheme?

Comment (satishkamatam) [reply app/views/application/scheme/chooseyourschemes.scala.html:58]: we have a java script which applies red background to the required qualifications if the selected scheme is special (means requires A-Level/ Stem level). This is how it is called in css as well as javascript.

**Pull request merged**